src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,8 +1057,8 @@ def customizePerJob(self, job):
         if job.get('taskName', None):
             jdl.append('+WMAgent_SubTaskName = "%s"\n' % job['taskName'])
 
-        if job.get('taskType', None):
-            jdl.append('+CMS_JobType = "%s"\n' % job['taskType'])
+        if job.get('subTaskType', None):
```

Comment:
Same with this one, it was pushed recently by:
https://github.com/dmwm/WMCore/commit/a42abf7638b87b064bbae9e6dcde1d9c48a8a49b#diff-9b070481bcf1072545075872a49c7968